src/crud/useArtifactCategory.ts

```diff
@@ -37,8 +42,15 @@ export const useArtifactCategory = () => {
     return fromLocal[val] || val;
   };
 
-  const getArtifactCategorys = () => {
+  const getArtifactCategorys = async () => {
     const categorys: IArtifactCategory[] = [];
+    /* wait for new categories remote id to fill in */
+    await waitForIt(
```

Comment:
This probably won't work offline. It should be conditioned by (!offline)

Reply:
They can't add a category if offline